content/pages/technical-workshop-series.md

```diff
@@ -0,0 +1,13 @@
+Title: Technical Workshop Series
+Status: hidden
+
+Our Technical Workshop Series is a series of three technical skill workshops
+that we will be hosting over the course of the next three terms.
+
+These events are generously sponsored by [BlackBerry](http://ca.blackberry.com)!
+BlackBerry has agreed to fund our three events and we are very excited to 
+work with them!
+
+To view all the event posts for our technical workshops, click 
+[here](http://wics.uwaterloo.ca/tags/blackberry). In Fall 2015 we ran our 
+Arduino Workshop, and we plan on running one more this term.
```

Comment:
maybe specify 'this term' since there's no date associated with this post? or do you want to change it later?